client/src/state/Run/RunAPI.ts

```diff
@@ -3,16 +3,18 @@ import superagent from 'superagent';
 import { IRunState } from './RunTypes';
 
 export async function getRuns(): Promise<IRunState> {
-    return (await superagent.get(`/api/v1/run`)).body;
+    const body = (await superagent.get(`/api/run/api/v1/run`)).body;
+
+    if (body === null) {
+        throw new Error('invalid body');
+    }
+    return body;
 }
 
-export async function upload(): Promise<IRunState> {
-    return (await superagent.post(`/api/v1/run/upload`)).body;
+export async function upload(contentId?: string): Promise<IRunState> {
+    return (await superagent.post(`/api/run`).send({ contentId })).body;
```

Comment:
Why does the URL look differently from the others?

Reply:
proxy and non proxy request.. :D